.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Create a report to help us improve the ohsome API.
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+### Bug Description
+A clear and concise description of what the bug is.
+
+### General Information
+ - Affected endpoint(s) [e.g. /contributions/count]
+ - Affected parameter(s) [e.g. bpolys, format]
+ - HTTP method used [e.g. POST]
+ - Used tool/library for the request [e.g. cURL, Postman]
+
+### How to Reproduce
+Steps [e.g.] to reproduce the behaviour:
+1. Start locally the ohsome API
+2. Send this request [URL] (http://localhost:8080/contributions/count?bboxes=8.67%2C49.39%2C8.71%2C49.42&time=2017-01-01%2F2017-01-02%2FP1Y)
+3. See error message 'Unable to find...'
+
+### Expected Behaviour
+A clear and concise description of what you expected to happen [e.g. "The error message should be..."]
+
+### Further Information
+
+#### Error messages, logs, screenshots
+If applicable, add printed error messages, log files or screenshots to help explain your problem.
+
+#### Additional information
+Add any other further information about the problem here.
+
+### System Information
+Please complete the following information:
+ - OS: [e.g. Ubuntu 20.04 LTS]
+ - Java Version: [e.g. openjdk version "11.0.9.1"]
+ - ohsome API Version: [e.g. 1.4.0]
+ - Maven version: [e.g. 3.6.3]
```

Comment:
```suggestion
```

This is generally irrelevant for ohsome API issues, so I would not ask for it in every bug report. If needed, the users can add such information in the _additional information_ section.